float64[] pos_gains
float64[] vel_gains
float64 qd_filt
---

